test failure of acceptance_test.py/test_manytests_to_one_import_error

Hi,
running the testsuite of pytest-xdist I've got a failure of *testing/acceptance_test.py::TestDistribution::test_manytests_to_one_import_error* (both, Python 2.7 and 3.5):
```
$ python2.7 -m pytest -v -x
{...}
testing/acceptance_test.py::TestDistribution::test_manytests_to_one_import_error FAILED
=============================================== FAILURES ================================================
__________________________ TestDistribution.test_manytests_to_one_import_error __________________________

self = <acceptance_test.TestDistribution instance at 0x7f1c66782f38>
testdir = <Testdir local('/tmp/pytest-of-aham/pytest-10/testdir/test_manytests_to_one_import_error0')>

    def test_manytests_to_one_import_error(self, testdir):
        p1 = testdir.makepyfile("""
                import __import_of_missing_module
                def test_import():
                    pass
            """)
        result = testdir.runpytest(p1, '--tx=popen', '--tx=popen')
>       assert result.ret == 1
E       assert 2 == 1
E        +  where 2 = <_pytest.pytester.RunResult instance at 0x7f1c66728a70>.ret

/tmp/sandbox244/testing/acceptance_test.py:65: AssertionError
{...}
==================================== ERRORS ====================================
____________ ERROR collecting test_manytests_to_one_import_error.py ____________
ImportError while importing test module '/tmp/pytest-of-aham/pytest-10/testdir/test_manytests_to_one_import_error0/test_manytests_to_one_import_error.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
test_manytests_to_one_import_error.py:1: in <module>
    import __import_of_missing_module
E   ImportError: No module named __import_of_missing_module
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
```
The import error is intended, though.
DS

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,26 +5,28 @@ python:
 install: pip install -U tox setuptools_scm
 env:
   matrix:
-  - TESTENV=py26-pytest26
-  - TESTENV=py26-pytest27
-  - TESTENV=py26-pytest28
-  - TESTENV=py26-pytest29
-  - TESTENV=py27-pytest26
-  - TESTENV=py27-pytest27
-  - TESTENV=py27-pytest28
-  - TESTENV=py27-pytest29
-  - TESTENV=py34-pytest26
-  - TESTENV=py34-pytest27
-  - TESTENV=py34-pytest28
-  - TESTENV=py34-pytest29
-  - TESTENV=py35-pytest27
-  - TESTENV=py35-pytest28
-  - TESTENV=py35-pytest29
-  - TESTENV=py27-pytest28-pexpect
-  - TESTENV=py35-pytest28-pexpect
-  - TESTENV=flakes
-  - TESTENV=readme
-script: tox --recreate -e $TESTENV
+  # note: please use "tox --listenvs" to populate the build matrix
+  - TOXENV=py26-pytest27
+  - TOXENV=py26-pytest28
+  - TOXENV=py26-pytest29
+  - TOXENV=py26-pytest30
+  - TOXENV=py27-pytest27
+  - TOXENV=py27-pytest28
+  - TOXENV=py27-pytest29
+  - TOXENV=py27-pytest30
+  - TOXENV=py34-pytest27
+  - TOXENV=py34-pytest28
+  - TOXENV=py34-pytest29
+  - TOXENV=py34-pytest30
+  - TOXENV=py35-pytest27
+  - TOXENV=py35-pytest28
+  - TOXENV=py35-pytest29
+  - TOXENV=py27-pytest28-pexpect
+  - TOXENV=py35-pytest28-pexpect
+  - TOXENV=flakes
+  - TOXENV=readme
+
+script: tox
 notifications:
   irc:
     channels:

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,3 +1,8 @@
+1.16.0
+------
+
+- ``pytest-xdist`` now requires pytest 2.7 or later.
+
 1.15.0
 ------
 

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,3 +1,26 @@
+environment:
+  matrix:
+  # note: please use "tox --listenvs" to populate the build matrix
+  - TOXENV: "py26-pytest27"
+  - TOXENV: "py26-pytest28"
+  - TOXENV: "py26-pytest29"
+  - TOXENV: "py26-pytest30"
+  - TOXENV: "py27-pytest27"
+  - TOXENV: "py27-pytest28"
+  - TOXENV: "py27-pytest29"
+  - TOXENV: "py27-pytest30"
+  - TOXENV: "py34-pytest27"
+  - TOXENV: "py34-pytest28"
+  - TOXENV: "py34-pytest29"
+  - TOXENV: "py34-pytest30"
+  - TOXENV: "py35-pytest27"
+  - TOXENV: "py35-pytest28"
+  - TOXENV: "py35-pytest29"
+  - TOXENV: "py27-pytest28-pexpect"
+  - TOXENV: "py35-pytest28-pexpect"
+  - TOXENV: "flakes"
+  - TOXENV: "readme"
+
 install:
   - C:\Python35\python -m pip install tox setuptools_scm
 

--- a/setup.py
+++ b/setup.py
@@ -20,7 +20,7 @@ setup(
         ],
     },
     zip_safe=False,
-    install_requires=['execnet>=1.1', 'pytest>=2.4.2', 'py>=1.4.22'],
+    install_requires=['execnet>=1.1', 'pytest>=2.7.0', 'py>=1.4.22'],
     setup_requires=['setuptools_scm'],
     classifiers=[
         'Development Status :: 5 - Production/Stable',

--- a/testing/acceptance_test.py
+++ b/testing/acceptance_test.py
@@ -62,7 +62,7 @@ class TestDistribution:
                 pass
         """)
         result = testdir.runpytest(p1, '--tx=popen', '--tx=popen')
-        assert result.ret == 1
+        assert result.ret in (1, 2)
         result.stdout.fnmatch_lines([
             "E   ImportError: *__import_of_missing_module*",
         ])

--- a/tox.ini
+++ b/tox.ini
@@ -1,8 +1,7 @@
 [tox]
 # if you change the envlist, please update .travis.yml file as well
 envlist=
-  py{26,27,34}-pytest2{6,7,8,9}
-  py35-pytest2{7,8,9}
+  py{26,27,34,35}-pytest{27,28,29,30}
   py{27,35}-pytest28-pexpect
   flakes
   readme  
@@ -15,10 +14,10 @@ deps =
   pycmd
   # to avoid .eggs
   setuptools_scm
-  pytest26: pytest~=2.6.1
   pytest27: pytest~=2.7.2
   pytest28: pytest~=2.8.7
-  pytest29: pytest~=2.9.1
+  pytest29: pytest~=2.9.2
+  pytest30: pytest~=3.0.5
   pexpect: pexpect
 platform=
   pexpect: linux|darwin    

